app/views/account/tenants/index.html.slim

```diff
@@ -0,0 +1,8 @@
+h1 All tenants
+= link_to "Add new tenant", new_account_tenant_path, class: 'btn btn-primary'
+
+ul.tenant
```

Comment:
remove this class "ul.tenant". leave just "ul"

Reply:
done